src/generate.js

```diff
@@ -2,39 +2,177 @@ import prefixAll from 'inline-style-prefix-all';
 
 import {
     objectToPairs, kebabifyStyleName, recursiveMerge, stringifyValue,
-    importantify
+    importantify, uniquify, mapObj,
 } from './util';
 
+/**
+ * Generate a map from descendant selector names to all of the classnames that
+ * could be used for that selector.
+ *
+ * In StyleSheet.create, we add a special `_names` object to each place where a
+ * descendant style is used containing the classname associated with that
+ * descenant. Here, we traverse the styles and make a map from the descendant
+ * selectors to all of the classnames in those `_names` objects. (Since we
+ * merge together many styles that come out of StyleSheet.create, and we store
+ * the classnames as keys on the `_names` object, the classnames that we want
+ * all end up as keys on the final merged object.)
+ *
+ * For example, given styles looking like:
+ * ```
```

Comment:
Since this is a private API, and therefore unlikely to be nicely presented as HTML docs, I don't think we need markdown indicators. (If you do want to keep them, you're missing a closing triple backtick before the `@returns` below)